modules/ROOT/pages/_partials/api-gov-concepts.adoc

```diff
@@ -47,87 +47,19 @@ MuleSoft provides several rulesets in Exchange, such as Anypoint API Best Practi
 
 [[governed-apis]]
 governed APIs::
-APIs are governed if they are identified by the selection criteria of at least one of the governance profiles. If an API is governed, all versions of that API are considered one governed API. Subscription limits are set based on your organization's purchased capacity and the UI gives information about usage and alerts when you are nearing or exceeding your subscription capacity. 
+APIs are governed if they are identified by the selection criteria of at least one governance profile. If an API is governed, all versions of that API are considered one governed API. Subscription limits are set based on your organization's purchased capacity and the UI gives information about usage and alerts when you are nearing or exceeding your subscription capacity. 
```

Comment:
Any way to make more active?

Reply:
The term "governed APIs" is tied to usage aka billing, so I prefer to keep that use of the word here. It would also overcomplicate the definition to try to make it completely active.

Reply:
OK, but there are other instances of passive voice in this bit.